HABApp/openhab/http_connection.py

```diff
@@ -320,6 +331,35 @@ async def async_get_item(self, item_name: str) -> dict:
         else:
             return await ret.json(encoding='utf-8')
 
+    async def async_thing_remove_link(self, thing_uid: str, channel: str, item_name: str) -> bool:
+        if self.config.general.listen_only:
+            return False
+
+        fut = self.__session.delete(self.__get_link_url(thing_uid, channel, item_name), json={})
```

Comment:
Is the parameter json really required?
```suggestion
        fut = self.__session.delete(self.__get_link_url(thing_uid, channel, item_name))
```

Reply:
nope, removed